roles/ceph-rgw-loadbalancer/templates/keepalived.conf.j2

```diff
@@ -30,7 +30,7 @@ vrrp_instance {{ instance['name'] }} {
     }
 {% if instance['virtual_route_source'] and instance['virtual_route_gateway'] %}
     virtual_routes {
-  {%- for source in instance['virtual_route_source'] %}
+  {%- for source in instance['virtual_route_source'] %} 
```

Comment:
Don't ask me why but the space is necessary or else the template does not come our properly.